pkgs/applications/misc/insulator2/default.nix

```diff
@@ -11,7 +12,7 @@
 , perl
 , cyrus_sasl
 , stdenv
-, fixup_yarn_lock
+, fixup-yarn-lock
```

Comment:
```suggestion
```

This can probably just be removed. It's already pulled in by yarnConfigHook and not ran anywhere else here

Reply:
fixed